src/core/pointcloud/qgspointcloudrenderer.cpp

```diff
@@ -230,8 +230,8 @@ void QgsPointCloudRenderer::restoreCommonProperties( const QDomElement &element,
 
   mMaximumScreenError = element.attribute( QStringLiteral( "maximumScreenError" ), QStringLiteral( "0.3" ) ).toDouble();
   mMaximumScreenErrorUnit = QgsUnitTypes::decodeRenderUnit( element.attribute( QStringLiteral( "maximumScreenErrorUnit" ), QStringLiteral( "MM" ) ) );
-  mPointSymbol = qgsEnumKeyToValue( element.attribute( QStringLiteral( "pointSymbol" ) ), Qgis::PointCloudSymbol::Square );
-  mDrawOrder2d = qgsEnumKeyToValue( element.attribute( QStringLiteral( "drawOrder2d" ) ), Qgis::PointCloudDrawOrder::Default );
```

Comment:
Can we not keep those lines as qgsEnumKeyToValue is trying int value if key is not matching?

Reply:
I'm not in favour of that -- it forces a needless break of project forward compatibility for little gain. (I realise forward compatibility isn't mandated, but that doesn't mean we should break things without a good reason)